db/schema.rb

```diff
@@ -1392,7 +1398,7 @@
     t.string "type", limit: 255, null: false
     t.string "name", limit: 255, null: false
     t.integer "parent_id"
-    t.serial "weight"
+    t.serial "weight", null: false
```

Comment:
This issue again: Don't change this line (remove the change) since this is not related to this PR.